src/shruti/FormData.java

```diff
@@ -0,0 +1,68 @@
+package shruti;
+
+	
+	import org.openqa.selenium.By;
+	import org.openqa.selenium.WebDriver;
+	import org.openqa.selenium.WebElement;
+	import org.openqa.selenium.chrome.ChromeDriver;
+
+	public class FormData {
+
+		// WebDriver driver = new ChromeDriver();
+
+		void dataEntryinForm() {
+			System.setProperty("webdriver.chrome.driver",
+					".\\Resources\\chromedriver.exe");
+			WebDriver driver = new ChromeDriver();
+			driver.get("file:///D:/Technocredit/Selenium%20recordings/@Selenium/SeleniumAssignment_1.html");
+			WebElement firstName = driver.findElement(By.id("first name"));
+			firstName.sendKeys("Shruti");
+			WebElement lastName = driver.findElement(By.id("last name"));
+			lastName.sendKeys("Dubey");
+			WebElement email = driver.findElement(By.id("E-mail"));
+			email.sendKeys("shrutidubey@gmail.com");
+			WebElement companyName = driver.findElement(By.id("Company Name"));
+			companyName.sendKeys("Infogain");
+
+			WebElement gender = driver.findElement(By.id("maleG"));
```

Comment:
You Swap the validation, for Gender radio button if ‘Female’ radio button is by default selected then select ‘Male’ radio button and vice versa.

Reply:
Changes done as per comment
![image](https://user-images.githubusercontent.com/66841149/90503110-8b6b3d00-e16c-11ea-88be-dde9cca712c2.png)